pages/challenges.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import Layout from '../components/Layout'
+import LessonTitleCard from '../components/LessonTitleCard'
+
+const setupAlert = (
+  <div className="alert bg-primary text-white col-12 mt-4" role="alert">
```

Comment:
why not use the alert provided by bootstrap? https://getbootstrap.com/docs/4.4/components/alerts/